Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.8-alpine
+FROM alpine:3.14
```

Comment:
Just a small thing, but apparently, the `ruby:2.6.8-alpine` points to alpine:3.14 under the hood.

What do you think about us using the more explicit `ruby:2.6.8-alpine` name here?